commands/core/cli.drush.inc

```diff
@@ -1,25 +1,9 @@
 <?php
 
-use Drush\Log\LogLevel;
-use Drupal\Component\Assertion\Handle;
-use Drush\Psysh\DrushHelpCommand;
-use Drush\Psysh\DrushCommand;
-use Drush\Psysh\Shell;
-
 /**
  * Implements hook_drush_command().
  */
 function cli_drush_command() {
-  $items['core-cli'] = array(
-    'description' => 'Open an interactive shell on a Drupal site.',
-    'remote-tty' => TRUE,
```

Comment:
'remote-tty' causes `drush_do_command_redispatch()` to add `#tty` to the backend options, which causes backend invoke to add -t to the ssh session.  I think that is critical to the remote operation of the shell.